tests/test.sh

```diff
@@ -654,6 +659,8 @@ if [[ "$MBTILES_BIN" != "-" ]]; then
   set -x
 
   $MBTILES_BIN summary ./tests/fixtures/mbtiles/world_cities.mbtiles 2>&1 | tee "$TEST_OUT_DIR/summary.txt"
+  $MBTILES_BIN summary --format json-pretty ./tests/fixtures/mbtiles/world_cities.mbtiles 2> "$TEST_OUT_DIR/summary.pretty.json.err" | cleanup_json | tee "$TEST_OUT_DIR/summary.pretty.json"
+  $MBTILES_BIN summary --format json ./tests/fixtures/mbtiles/world_cities.mbtiles 2> "$TEST_OUT_DIR/summary.json.err" | cleanup_json | tee "$TEST_OUT_DIR/summary.json"
```

Comment:
In no other place do we not log the `.err` log.
Is adding it intentional?

-> I think we should only add this snapshot if we expect something to be therere, or?